shared-helpers/src/multiselectQuestions.tsx

```diff
@@ -130,10 +130,14 @@ export const getRadioFields = (
         fieldClassName="ml-0"
         type={"radio"}
         name={fieldName(question?.text, applicationSection)}
-        error={errors && errors[question?.text]}
+        error={
+          errors &&
+          Object.keys(errors).length > 0 &&
+          errors["application"]["programs"][question?.text]
```

Comment:
This could be condensed to `errors && errors.application?.programs?.[question?.text]`